concepticondata/references/references.bib

```diff
@@ -2099,3 +2099,16 @@ @Phdthesis{Yang2011
   Note                                = {PhD Thesis},
   Address                             = {Bundoora}
 }
+
+@Article{Vejdemo2016,
+  Title                               = {{S}emantic {F}actors {P}redict the {R}ate of {L}exical {R}eplacement of {C}ontent {W}ords},
```

Comment:
we usually never capitalize the text, only if its place names, etc., so the {} brackets should be removed here.